### ISSUE TITLE
Firebase Performance Monitoring Console/Logcat Spam

### ISSUE THREAD

**Issue by findyourexit** [firebase/firebase-android-sdk#3441]
* **Android Studio version:** Android Studio Bumblebee | 2021.1.1 Patch 1 (Build #AI-211.7628.21.2111.8139111)
  * **Firebase Component:** Firebase Performance Monitoring
  * **Component version:** `com.google.firebase:firebase-bom:29.1.0` (`com.google.firebase:firebase-perf:20.0.5`, `com.google.firebase:perf-plugin:1.4.1`)

#### Steps to reproduce:

The steps to reproduce this were rather simple. To reproduce, I simply:
1) Followed the official [getting started guide](https://firebase.google.com/docs/perf-mon/get-started-android) for Android to add Firebase Performance Monitoring to my app.
2) Built/ran my app using a physical device deployment target (Pixel 5 running Android 12).
3) Watched Logcat/console and observed the same two lines (with very minor variation) appearing over and over again, without end.

The two rows that keep appearing are as follows:
```
2022-02-14 16:17:49.179 25682-25745/com.redacted.app.redacted.dev D/TransportRuntime.SQLiteEventStore: Storing event with priority=DEFAULT, name=FIREPERF for destination cct
2022-02-14 16:17:49.182 25682-25745/com.redacted.app.redacted.dev D/TransportRuntime.JobInfoScheduler: Upload for context TransportContext(cct, DEFAULT, MSRodHRwczovL2ZpcmViYXNlbG9nZ1luZy1wYS5nb59nbGVhcGlzLmNvbS92MS9maXJlbG9nL2xlZ2FjeS9iYXRjaGxvZ2xBSXphU3lDY2traUg4aTJaQVJ3T3MxTEV6RktsZDE1YU3HOG96S28=) is already scheduled. Returning...
```

As mentioned, the above rows continue to appear over and over, with approximately less than a second between each occurrence.

I've found loosely related posts on StackOverflow and on [other projects here on GitHub](https://github.com/FirebaseExtended/flutterfire/issues/6140), but no resolution seems apparent.

In order to pre-empt the recommendation to disable logging via `AndroidManifest.xml` `firebase_performance_logcat_enabled` `meta-data`, I've already done so, however it doesn't stop the above from occurring:

```xml
<meta-data
    android:name="firebase_performance_logcat_enabled"
    android:value="false" />
```

Any help with this would be greatly appreciated!

**Comment by visumickey** [comment]
While we wait on reproducing this, an update is that these messages are coming from the library Firebase Performance uses for event dispatching. We are working with them to figure out ways to having a good logging message.

Reference: https://github.com/firebase/firebase-android-sdk/blob/master/transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/AlarmManagerScheduler.java#L107

**Comment by findyourexit** [comment]
Thanks for the super prompt replies @argzdev and @visumickey!

@argzdev - I'll endeavour to produce an MCVE to share here. When/if I can do so, I'll report back right away.

**Comment by ktiniatros** [comment]
Hello,

I created a reproducable case: https://issuetracker.google.com/action/issues/225529530/attachments/36592621?download=true

Download, unzip and open the project in android studio and run it on any emulator. You will see the spam in logcat.

Since you are working on this, please remove the logs or ideally make them configurable (be default to not show with an option to enable them).

**Comment by jeremyjiang-dev** [comment]
@vkryachko Would you like to take a look at this as the logs are coming from the firelog side?

**Comment by russellwheatley** [comment]
Hey all 👋 , I thought I would chime in to let you know that this is an [issue](https://github.com/firebase/flutterfire/issues/6140) for a number of users in FlutterFire which thinly wraps around this SDK. It does appear to be an intermittent issue, but a very real one. 

For another possible, reproducible sample, [see here](https://github.com/firebase/flutterfire/issues/6140#issuecomment-877841159).

**Comment by vkryachko** [comment]
@jeremyjiang-dev These logs happen any time an event is scheduled for upload, I think an initial question to answer is whether that's normal for FirePerf to log this much data? It sounds like that should not be the case, can you check if you're uploading more data than necessary?

**Comment by jeremyjiang-dev** [comment]
Hey @vkryachko ! We have not changed the volume or frequency on our recent SDKs and the QPS looks normal. Is there a way to turn off the Firelog logs?

**Comment by tomkulaga** [comment]
I wanted to check if, has there been any updates?
